src/java.base/share/classes/sun/net/www/protocol/https/HttpsClient.java

```diff
@@ -148,14 +146,14 @@
             ciphers = null;
         } else {
             StringTokenizer     tokenizer;
-            Vector<String>      v = new Vector<String>();
+            ArrayList<String>   v = new ArrayList<>();
 
             tokenizer = new StringTokenizer(cipherString, ",");
             while (tokenizer.hasMoreTokens())
-                v.addElement(tokenizer.nextToken());
+                v.add(tokenizer.nextToken());
             ciphers = new String [v.size()];
```

Comment:
Looks like this whole `else` block can be simplified to `ciphers = cipherString.split(",");`

Reply:
It's not a drop-in replacement. Result is different for some Strings. For example for `,  A`
I would prefer to preserve existing behavior under this cleanup.

Reply:
Then let's keep it as is